eng/common/scripts/Create-APIReview.ps1

```diff
@@ -164,6 +164,10 @@ function IsApiviewStatusCheckRequired($packageInfo)
     if (($packageInfo.SdkType -eq "client" -or $packageInfo.SdkType -eq "spring") -and $packageInfo.IsNewSdk) {
         return $true
     }
+
+    if (($packageInfo.SdkType -eq "mgmt") -and ($Language -eq "dotnet")) {
```

Comment:
This will work. We wanted to change this to be driven by language level repo instead of common script. Can we have a property in Language-Settings that set to true in .NET repo and let the default be false if not present. For e.g.  `Is-API-Approval-Required-For-Mgmt`

Reply:
Instead of a flag can we have a filter function that can be overridden by a given language. If they don't override it then fallback to this logic.